rbx_dom_lua/src/database.json

```diff
@@ -24635,7 +24759,7 @@
           "Enum": 0
         },
         "ScriptGuid": {
-          "String": "{7EFC8201-25F5-42C7-8DDB-DD7BD2D32FE7}"
+          "String": "{7604441B-B379-4EE1-984B-AE157BC63C77}"
```

Comment:
We should probably do something about ScriptGuid going forward because it seems like it isn't stable between versions... Which is probably expected behavior given it's a GUID.

It doesn't need to and shouldn't happen in this PR but it's something to think about because I'm not sure how to actually solve a property that we shouldn't store a default for like this.

Reply:
Agreed that it doesn't make sense to have a default ScriptGuid - we ought to catalogue any properties for which defaults aren't sensible and figure out a good strategy to deal with them